test_zero_even_op failed, libcaffe2_detectron_custom_ops_gpu.so not compiled

after make ops, and run python2 test_zero_even_op.py. I Got
AssertionError: Custom ops lib not found at '/home/workplace/human_pose/densePose/densepose/build/libcaffe2_detectron_custom_ops_gpu.so'

From CmakeLists.txt,  I wonder where the Flag CAFFE2_USE_CUDA is setted? 
``
if (${CAFFE2_USE_CUDA})
  list(APPEND CUDA_INCLUDE_DIRS -I${CAFFE2_INCLUDE_DIRS})
  CUDA_ADD_LIBRARY(
      caffe2_detectron_custom_ops_gpu SHARED
      ${CUSTOM_OPS_CPU_SRCS}
      ${CUSTOM_OPS_GPU_SRCS})

  target_link_libraries(caffe2_detectron_custom_ops_gpu caffe2_gpu_library)
  install(TARGETS caffe2_detectron_custom_ops_gpu DESTINATION lib)
endif()
``
+ I have passed the cafffe and gpu test with the following scripts:
python2 -c 'from caffe2.python import core' 2>/dev/null && echo "Success" || echo "Failure"
Success
 python2 -c 'from caffe2.python import workspace; print(workspace.NumCudaDevices())'
1

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -38,7 +38,7 @@ target_link_libraries(caffe2_detectron_custom_ops caffe2_library)
 install(TARGETS caffe2_detectron_custom_ops DESTINATION lib)
 
 # Install custom GPU ops lib, if gpu is present.
-if (${CAFFE2_USE_CUDA})
+if (${CAFFE2_USE_CUDA} OR ${CAFFE2_FOUND_CUDA})
   # Additional -I prefix is required for CMake versions before commit (< 3.7):
   # https://github.com/Kitware/CMake/commit/7ded655f7ba82ea72a82d0555449f2df5ef38594
   list(APPEND CUDA_INCLUDE_DIRS -I${CAFFE2_INCLUDE_DIRS})

